shared/chat/conversation/messages/text.desktop.js

```diff
@@ -111,12 +112,22 @@ const _avatarStyle = {
   marginRight: globalMargins.tiny,
 }
 
-export default withHandlers({
-  onIconClick: (props: Props) => event => {
-    props.onAction(props.message, event)
-  },
-})(_MessageTextComponent)
-
 const stylesFirstNewMessage = {
   borderTop: `solid 1px ${globalColors.orange}`,
 }
+
+export default compose(
+  shouldUpdate((props: Props, nextProps: Props) => {
```

Comment:
I pondered this for awhile and I think we could do this short term , but i think longer term we might want to think about how we do styling differently.